cli/cmd/install_test.go

```diff
@@ -488,6 +488,8 @@ func TestValidate(t *testing.T) {
 			expectedError string
 		}{
 			{"valid", ""},
+			{"valid-with-rsa-anchor", ""},
+			{"too-weak-rsa-anchor", ""},
```

Comment:
Removing this line should fix the unit tests
```suggestion
```

Reply:
Of course! I thought I had submitted that line in my last try, but it seems I didn't finally push it. Thanks a lot @alpeb